anonymizer/anonymizer.c

```diff
@@ -212,7 +212,7 @@ void ip_anonymize(void *field_ptr, uint8_t mode)
 char *string_anonymize(void *field_ptr, uint32_t field_len, uint8_t mode, regex_t regex_IPV4, regex_t regex_IPV6)
 {
    int reti;
-   regmatch_t ip;
+   regmatch_t ip[2]; // expecting MAX 1 occurence, see man regexec for more info
```

Comment:
`2` should be rather a (cpp macro) constant defined on one place - it improves readability and makes it easier to change this value just on one place in the future... on L225 and L227 there is the same constant with the same meaning.

Reply:
Additionally, why 2? Why not just 1? (as it was...)